contrib/python/api/skydive/websocket/client.py

```diff
@@ -19,19 +19,29 @@
 # under the License.
 #
 
-import asyncio
+try:
+    import asyncio
+except ImportError:
+    import trollius as asyncio
 import base64
+import functools
 import json
 import http.client as httplib
```

Comment:
need to address import issue python2/python3 ?

```
try:
    import http.client
except ImportError:
    import httplib
```

Reply:
Will fix